cmake/Tools/layout_tool.py

```diff
@@ -135,9 +135,12 @@ def _validate_remote_ap(input_remote_ip, input_remote_connect, remote_on_check):
         else:
             # Validation steps based on the asset mode
             if asset_mode == ASSET_MODE_PAK:
+                if platform_name_lower == 'ios':
+                    project_asset_path = pathlib.Path(project_path) / 'AssetBundling/Bundles'
```

Comment:
why is this specific to ios?

Reply:
add to this question, why is it specific to ios AND pak files?